test/unit_tests/modules/nav-primary-spec.js

```diff
@@ -45,61 +47,141 @@ describe( 'Get Event States', function() {
-      $( '.nav-link-1' ).trigger( 'click' );
-
       expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'true' );
     } );
 
     it( 'should not toggle a non-sibling sub nav', function() {
-      $( '.nav-link-2' ).trigger( 'click' );
+      expect( $( '.sub-nav-2' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+    } );
 
-      expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+    it( 'should focus the first link in the sibling sub nav', function( done ) {
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
+        expect( $link.is( ':focus' ) ).to.be.true;
+        done();
+      }, 500 );
     } );
-  } );
 
-  describe( 'Primary Nav Link Events - One expanded', function() {
-    it( 'should close itself', function() {
-      $( '.sub-nav-1' ).attr( 'aria-expanded', 'true' );
-      $( '.nav-link-1' ).trigger( 'click' );
+    it( 'should not focus the first link in a non-sibling sub nav', function( done ) {
+      var $nextlink = $( $( '.sub-nav-2' ).find( 'a' )[0] );
 
-      expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
+        expect( $nextlink.is( ':focus' ) ).to.be.false;
+        done();
+      }, 500 );
     } );
+  } );
+
+  describe( 'Primary Nav Link Events - One expanded', function() {
+    beforeEach( function() {
+      $link = $( $( '.sub-nav-2' ).find( 'a' )[0] );
 
-    it( 'should close others, then open sibling sub nav', function() {
       $( '.sub-nav-1' ).attr( 'aria-expanded', 'true' );
       $( '.nav-link-2' ).trigger( 'click' );
+    } );
 
+    it( 'should close others', function() {
       expect( $( '.sub-nav-1' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+    } );
 
+    it( 'should open the sibling sub nav', function( done ) {
       setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
         expect( $( '.sub-nav-2' ).attr( 'aria-expanded' ) ).to.equal( 'true' );
+        done();
+      }, 500 );
+    } );
+
+    it( 'should focus the first link in sibling sub nav', function( done ) {
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
+        expect( $link.is( ':focus' ) ).to.be.true;
+        done();
+      }, 500 );
+    } );
+
+    it( 'should close itself', function( done ) {
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
+        $( '.nav-link-2' ).trigger( 'click' );
+
+        expect( $( '.sub-nav-2' ).attr( 'aria-expanded' ) ).to.equal( 'false' );
+        done();
+      }, 500 );
+    } );
+
+    it( 'should focus the primary link after closing', function( done ) {
+      setTimeout( function() { // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
```

Comment:
No, it's because we're expanding it in the beforeEach (https://github.com/cfpb/cfgov-refresh/pull/971#discussion-diff-39461028R90), and we have to wait for that to finish, then we have to wait for it to close to test the state.